lib/services/workflow-api-service.js

```diff
@@ -155,7 +236,22 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.findOne({
+            node: target,
+            _status: Constants.Task.ActiveStates
+        });
+    };
+
+    WorkflowApiService.prototype.getActiveWorkflows = function(query) {
```

Comment:
I'm not sure I follow your concern.  Are you suggesting that _graph_ be replaced with _workflow_?  If so, I agree and I think this applies to many of the functions in this service.  I'd suggest adding an issue and cleaning up in the future.